errors.go

```diff
@@ -42,13 +42,13 @@ var _ APIError = (*GenericAPIError)(nil)
 // OperationError decorates an underlying error which occurred while invoking
 // an operation with names of the operation and API.
 type OperationError struct {
-	ServiceID     string
+	ClientID      string
 	OperationName string
 	Err           error
 }
 
 // Service returns the name of the API service the error occurred with.
-func (e *OperationError) Service() string { return e.ServiceID }
+func (e *OperationError) Service() string { return e.ClientID }
```

Comment:
Does it make sense to refactor the `OperationError.Service` method to be `Client` instead of service? or is it better to stick with `Service` in this context?

Reply:
Ah this was a miss when I auto-renamed, I missed that this method was named differently. I don't have a strong preference between `Service` vs `Client`, I just think we want to be consistent, and likely want to choose a name that is both generic and not Amazon specific. Maybe we should stick with `Service`?. As this matches with the smithy naming scheme of a `service` shape.

Reply:
ok, yeah lets stick with `service` naming.